portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,8 @@
 
 package com.google.sps.servlets;
 
+import java.util.*;
```

Comment:
It is better to import only the class you need. Use,
#import java.util.ArrayList.

Reply:
I don't the change. Please resolve after the change has been made